src/SwedbankPay.Sdk/Extensions/HttpRequestMessageExtensions.cs

```diff
@@ -3,7 +3,7 @@
 
 using Newtonsoft.Json;
 
-namespace SwedbankPay.Sdk
+namespace SwedbankPay.Sdk.Extensions
```

Comment:
Doesn't seem to be used anymore. Remove?

Reply:
I don't get this comment, would you kindly elaborate on this?

Reply:
Ah, you meant the entire file, I'll remove it.